benchmarks/recursive_computation/config.yaml

```diff
@@ -0,0 +1,50 @@
+shallow: &base
+  max_depth: 10
+  enable_sleep: false
+  sleep_duration: 0.1
+  nspans: 1
```

Comment:
Also, let's not think too much about the number of spans we add, and let's remove `nspans`